src/lib/Controller/api/supervisor.c

```diff
@@ -1062,16 +1081,16 @@ void wb_supervisor_set_label(int id, const char *text, double x, double y, doubl
 }
 
 void wb_supervisor_export_image(const char *filename, int quality) {
-  if (!robot_check_supervisor("wb_supervisor_export_image"))
+  if (!robot_check_supervisor(__FUNCTION__))
     return;
 
   if (!filename || !filename[0]) {
-    fprintf(stderr, "Error: wb_supervisor_export_image() called with NULL or empty 'filename' argument.\n");
+    fprintf(stderr, "Error: %s() called with NULL or empty 'filename' argument.\n", __FUNCTION__);
```

Comment:
Be careful you are doing double-work here. This is already fixed on develop.

Reply:
Oops... Too late...